src/SSHDebugPS/AD7Port.cs

```diff
@@ -216,5 +221,11 @@ public bool IsLinux()
         {
             return GetConnection(ConnectionReason.Deferred).IsLinux();
         }
+
+        public void Clean()
+        {
+            GetConnection(ConnectionReason.Deferred).Clean();
```

Comment:
>GetConnection(ConnectionReason.Deferred) [](start = 12, length = 40)

If `_connection` is null, should this actually be a nop?

Reply:
fixed